packages/babel-template/src/parse.js

```diff
@@ -58,10 +58,10 @@ export default function parseAndBuildMetadata<T>(
   };
 }
 
-function placeholderVisitorHandler(
-  node: BabelNode,
+function placeholderVisitorHandler<T: BabelNode, S: MetadataState>(
+  node: T,
```

Comment:
Since we're not using `T` anywhere, it doesn't seem like having this be a type parameter is needed. `BabelNode` is a parent class of every type of node, so accepting it as the type of the param should be fine.